src/ansys/fluent/core/launcher/pyfluent_enums.py

```diff
@@ -149,6 +134,19 @@ def _get_enum_map(self):
             self.GUI: ("",),
         }
 
+    def __lt__(self, other):
```

Comment:
It's used for `UIMode` then? Please could you explain it in the description?

Reply:
@seanpearsonuk, it is not required. The earlier usage is modified now.